src/libstd/io/stdio.rs

```diff
@@ -204,6 +204,30 @@ impl Stdin {
     ///
     /// For detailed semantics of this method, see the documentation on
     /// `BufRead::read_line`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io;
+    ///
+    /// #fn main() {
```

Comment:
This `fn main()` wrapper should be added automatically so there's no need to have this here or the indentation of the example.